src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -280,6 +280,11 @@ public String getQuestionSpecificEditFormHtml(int questionNumber) {
                 Slots.CONSTSUM_SELECTED_POINTS_PER_OPTION, pointsPerOption ? "selected" : "",
                 Slots.CONSTSUM_OPTION_TABLE_VISIBILITY, distributeToRecipients ? "style=\"display:none\"" : "",
                 Slots.CONSTSUM_POINTS, points == 0 ? "100" : Integer.toString(points),
+                Slots.OPTION_DISPLAY, distributeToRecipients ? "style=\"display:none\"" : "",
+                Slots.RECIPIENT_DISPLAY, distributeToRecipients ? "" : "style=\"display:none\"",
+                Slots.PER_OPTION_CHECKED, distributeToRecipients ? "" : pointsPerOption ? "checked" : "",
+                Slots.PER_RECIPIENT_CHECKED, distributeToRecipients ? pointsPerOption ? "checked" : "" : "",
```

Comment:
Not too fond of the double ternary conditional here, but it could be simplified to just

`distributeToRecipients && pointsPerOption ? "checked" : "",`